src/hivpy/experiment.py

```diff
@@ -1,34 +1,59 @@
 from dataclasses import dataclass
-
-from .population import Population
+from datetime import date, timedelta
 from .simulation import run_simulation, SimulationConfig
+import logging
+import os
 
+LEVELS = {
+    'DEBUG': logging.DEBUG,
+    'INFO': logging.INFO,
+    'WARNING': logging.WARNING,
+    'ERROR': logging.ERROR,
+    'CRITICAL': logging.CRITICAL
+}
 
 @dataclass
-class ExperimentConfig:
-    """Configuration parameters covering all stages of an experiment."""
-    population_size: int
-    simulation: SimulationConfig
```

Comment:
My initial intention for `ExperimentConfig` was that it would hold together all the configs we read from one file (for the simulation, the output, any post-processing etc),  that's why it was kept distinct from `SimulationConfig`. If you think that's not useful though, happy to remove it.

Reply:
Maybe a single Config class comprising both simulation and output config would be good - which would also be a place to have sensible default values in case none are provided in a config file.